INFO:root:----------------------------------------
INFO:root:Server loaded
INFO:root:200 POST /ws/usage/new (127.0.0.1) 1.16ms
INFO:root:----------------------------------------
INFO:root:Server loaded
INFO:root:new usage
INFO:root:{'_id': ObjectId('52ebdbb2848b0c7d9db52cdf'),
 u'date': datetime.datetime(2014, 1, 31, 17, 21, 54, 665000, tzinfo=<bson.tz_util.FixedOffset object at 0x7f0c2f077950>),
 u'duration': u'666',
 u'logType': u'3Dprinter',
 u'material quantity': u'0.3cm3',
 u'object': {u'extrusion_width_ratio': u'0',
             u'filament used': u'5.0mm',
             u'filament_diameter': u'2.92',
             u'file': u'gcode_example.gcode',
             u'fill_density': u'0.3',
             u'gcode-generation': u'generated by Slic3r 0.5.7 on 2011-11-28 at 07:27:59',
             u'infill_speed': u'30',
             u'layer_height': u'0.40',
             u'nozzle_diameter': u'0.5',
             u'perimeter_speed': u'30',
             u'perimeters': u'3',
             u'scale': u'1',
             u'solid_layers': u'3',
             u'travel_speed': u'100'},
 u'result': u'OK',
 u'tool': u'Test'}
INFO:root:200 POST /ws/usage/new (127.0.0.1) 1.31ms
INFO:root:----------------------------------------
INFO:root:----------------------------------------
INFO:root:Server loaded - 2014-01-31 18:27:51.026754
INFO:root:----------------------------------------
INFO:root:Server loaded - 2014-01-31 18:38:04.350252
INFO:root:new usage
INFO:root:{'_id': ObjectId('52ebdf81848b0c7e174d2e34'),
 u'date': datetime.datetime(2014, 1, 31, 17, 38, 9, 906000, tzinfo=<bson.tz_util.FixedOffset object at 0x7f47fe8949d0>),
 u'duration': u'666',
 u'logType': u'3Dprinter',
 u'material quantity': u'0.3cm3',
 u'object': {u'extrusion_width_ratio': u'0',
             u'filament used': u'5.0mm',
             u'filament_diameter': u'2.92',
             u'file': u'gcode_example.gcode',
             u'fill_density': u'0.3',
             u'gcode-generation': u'generated by Slic3r 0.5.7 on 2011-11-28 at 07:27:59',
             u'infill_speed': u'30',
             u'layer_height': u'0.40',
             u'nozzle_diameter': u'0.5',
             u'perimeter_speed': u'30',
             u'perimeters': u'3',
             u'scale': u'1',
             u'solid_layers': u'3',
             u'travel_speed': u'100'},
 u'result': u'OK',
 u'tool': u'Test'}
INFO:root:200 POST /ws/usage (127.0.0.1) 1.29ms
ERROR:root:Uncaught exception GET /ws/usage (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8888', method='GET', uri='/ws/usage', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Accept-Language': 'en-US,en;q=0.5', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:25.0) Gecko/20100101 Firefox/25.0', 'Connection': 'keep-alive', 'Cookie': 'PVELangCookie=fr; org.cups.sid=eb4c977f82cc5561dd1a1a446691198d', 'Test': 'test'})
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1042, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "./server.py", line 48, in get
    self.write(json.dump(usage.find()))
TypeError: dump() takes at least 2 arguments (1 given)
ERROR:root:500 GET /ws/usage (127.0.0.1) 0.89ms
WARNING:root:404 GET /favicon.ico (127.0.0.1) 0.20ms
WARNING:root:404 GET /favicon.ico (127.0.0.1) 0.15ms
INFO:root:----------------------------------------
INFO:root:Server loaded - 2014-01-31 18:39:40.368732
INFO:root:new usage
INFO:root:{'_id': ObjectId('52ebdfdf848b0c7ea8969b40'),
 u'date': datetime.datetime(2014, 1, 31, 17, 39, 43, 937000, tzinfo=<bson.tz_util.FixedOffset object at 0x7fdbf2ef19d0>),
 u'duration': u'666',
 u'logType': u'3Dprinter',
 u'material quantity': u'0.3cm3',
 u'object': {u'extrusion_width_ratio': u'0',
             u'filament used': u'5.0mm',
             u'filament_diameter': u'2.92',
             u'file': u'gcode_example.gcode',
             u'fill_density': u'0.3',
             u'gcode-generation': u'generated by Slic3r 0.5.7 on 2011-11-28 at 07:27:59',
             u'infill_speed': u'30',
             u'layer_height': u'0.40',
             u'nozzle_diameter': u'0.5',
             u'perimeter_speed': u'30',
             u'perimeters': u'3',
             u'scale': u'1',
             u'solid_layers': u'3',
             u'travel_speed': u'100'},
 u'result': u'OK',
 u'tool': u'Test'}
INFO:root:200 POST /ws/usage (127.0.0.1) 1.27ms
ERROR:root:Uncaught exception GET /ws/usage (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8888', method='GET', uri='/ws/usage', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Accept-Language': 'en-US,en;q=0.5', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:25.0) Gecko/20100101 Firefox/25.0', 'Connection': 'keep-alive', 'Cookie': 'PVELangCookie=fr; org.cups.sid=eb4c977f82cc5561dd1a1a446691198d', 'Cache-Control': 'max-age=0', 'Test': 'test'})
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1042, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "./server.py", line 48, in get
    self.write(json.dumps(usage.find()))
  File "/usr/lib/python2.7/json/__init__.py", line 233, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 203, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 266, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python2.7/json/encoder.py", line 180, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <pymongo.cursor.Cursor object at 0x7fdbf2a65fd0> is not JSON serializable
ERROR:root:500 GET /ws/usage (127.0.0.1) 1.26ms
ERROR:root:Uncaught exception GET /ws/usage (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:8888', method='GET', uri='/ws/usage', version='HTTP/1.1', remote_ip='127.0.0.1', body='', headers={'Accept-Language': 'en-US,en;q=0.5', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:25.0) Gecko/20100101 Firefox/25.0', 'Connection': 'keep-alive', 'Cookie': 'PVELangCookie=fr; org.cups.sid=eb4c977f82cc5561dd1a1a446691198d', 'Test': 'test'})
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1042, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "./server.py", line 48, in get
    self.write(json.dumps(usage.find()))
  File "/usr/lib/python2.7/json/__init__.py", line 233, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 203, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 266, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python2.7/json/encoder.py", line 180, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <pymongo.cursor.Cursor object at 0x7fdbf2a720d0> is not JSON serializable
ERROR:root:500 GET /ws/usage (127.0.0.1) 0.50ms
